code/frameworks/sveltekit/src/preview.ts

```diff
@@ -0,0 +1,79 @@
+import type { Decorator } from '@storybook/svelte';
+
+import { onMount } from 'svelte';
+import { setAfterNavigateArgument } from './mocks/app/navigation';
+import { setNavigating, setPage, setUpdated } from './mocks/app/stores';
+
+export const decorators: Decorator[] = [
+  (Story, ctx) => {
+    setPage(ctx.parameters?.sveltekit?.stores?.page);
+    setUpdated(ctx.parameters?.sveltekit?.stores?.updated);
+    setNavigating(ctx.parameters?.sveltekit?.stores?.navigating);
+    setAfterNavigateArgument(ctx.parameters?.sveltekit?.navigation?.afterNavigate);
+
+    onMount(() => {
```

Comment:
`onMount` won't be needed when https://github.com/storybookjs/storybook/pull/24921 has been merged right?

Reply:
they are great!